Formula/fastfec.rb

```diff
@@ -0,0 +1,25 @@
+class Fastfec < Formula
+  desc "Extremely fast FEC filing parser written in C"
+  homepage "https://github.com/washingtonpost/FastFEC"
+  url "https://github.com/washingtonpost/FastFEC/archive/refs/tags/0.0.3.tar.gz"
+  sha256 "13317652775d9734af52440d8b3fb3a0c759f94345b84ad7d2c80efdc18d5e0d"
+  license "MIT"
+
+  depends_on "pkg-config" => :build
+  depends_on "zig" => :build
+  depends_on "pcre"
+  uses_from_macos "curl"
+
+  def install
+    system "zig", "build", "-Dvendored-pcre=false", "--search-prefix", Formula["curl"].opt_prefix
```

Comment:
This won't work since there won't be anything at `Formula["curl"].opt_prefix`.

Does it work if you had `depends_on "curl"` instead of `uses_from_macos`?

Reply:
Realized the build errors were due to Zig's linker, and not being unable to find Curl (so revised to the old build script with a new build flag that fixes the linking issues)